conf/defaults.ini

```diff
@@ -1124,7 +1124,7 @@ license_path =
 # enable = feature1,feature2
 enable =
 
-# The new prometheus visual query builder 
+# The new prometheus visual query builder
```

Comment:
Note: I accidentally added a change to this file, which I've since removed. However, I can't get rid of this formatting fix. Not that it really changes anything.